sdk/core/src/serde/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod http_date;
```

Comment:
Not sure how I feel about having a `serde` module. Is there any reason this can't live in the date module?

Reply:
Done.